metricbeat/beat/list.go

```diff
@@ -0,0 +1,25 @@
+package beat
+
+// Make sure all active plugins are loaded
+// TODO: create a script to automatically generate this list
+import (
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/metricbeat/helper"
+
+	// List of all metrics to make sure they are registred
+	// Every new metric must be added here
+	_ "github.com/elastic/beats/metricbeat/module/beat"
+	_ "github.com/elastic/beats/metricbeat/module/elasticsearch/cluster-stats"
+	_ "github.com/elastic/beats/metricbeat/module/elasticsearch/info"
+	_ "github.com/elastic/beats/metricbeat/module/elasticsearch/stats"
+	_ "github.com/elastic/beats/metricbeat/module/redis/info"
```

Comment:
Another option here would be to just import the module and let the module import the metrics. Not sure which way is more maintainable, just figured I mention the option for discussion.